libs/shell/airdrop/src/lib/airdrop-page/airdrop-page.tsx

```diff
@@ -0,0 +1,77 @@
+import { useState } from 'react';
+import { ethToHaqq, useAddress } from '@haqq/shared';
+import { CaptchaModal, Container } from '@haqq/shell-ui-kit';
+import { AirdropEvm } from '../airdrop-evm/airdrop-evm';
+import { AirdropCosmos } from '../airdrop-cosmos/airdrop-cosmos';
+import { Address } from '../address/address';
+
+export function AirdropPage({
+  turnstileSiteKey,
+  airdropEndpoint,
+}: {
+  turnstileSiteKey?: string;
```

Comment:
Try to move into `Context` and use this context instead of passing props

Reply:
this is fast way to do it now. we refactor it later in another pr